src/PlatformServices/Services/TestDataSource.cs

```diff
@@ -47,11 +55,25 @@ public IEnumerable<object> GetData(UTF.ITestMethod testMethodInfo, ITestContext
 
             try
             {
+
+/* Unmerged change from project 'PlatformServices (net462)'
+Before:
                 this.GetConnectionProperties(testMethodInfo.GetAttributes<UTF.DataSourceAttribute>(false)[0], out providerNameInvariant, out connectionString, out tableName, out dataAccessMethod);
+After:
+                TestDataSource.GetConnectionProperties(testMethodInfo.GetAttributes<UTF.DataSourceAttribute>(false)[0], out providerNameInvariant, out connectionString, out tableName, out dataAccessMethod);
+*/
+
+/* Unmerged change from project 'PlatformServices (netstandard2.0)'
+Before:
+                this.GetConnectionProperties(testMethodInfo.GetAttributes<UTF.DataSourceAttribute>(false)[0], out providerNameInvariant, out connectionString, out tableName, out dataAccessMethod);
+After:
+                TestDataSource.GetConnectionProperties(testMethodInfo.GetAttributes<UTF.DataSourceAttribute>(false)[0], out providerNameInvariant, out connectionString, out tableName, out dataAccessMethod);
+*/
+                GetConnectionProperties(testMethodInfo.GetAttributes<UTF.DataSourceAttribute>(false)[0], out providerNameInvariant, out connectionString, out tableName, out dataAccessMethod);
             }
             catch (Exception ex)
             {
-                throw ex;
+                throw new Exception("Failed to get connection properties!", ex);
```

Comment:
It's usually not recommended (feels offensive) to have the error message ending with exclamation mark. Also if you are throwing an exception, I expect the message to be coming from a resource file (to be translatable).